src/arcaflow_plugin_sdk/test_schema.py

```diff
@@ -2117,6 +2161,172 @@ def test_one_of_inline(self):
             json_schema,
         )
 
+    def test_oneof_int_inline(self):
+        scope = schema.ScopeType(
+            {},
+            IntInlineUnion.__name__,
+        )
+        inline_int_key = 1
+        inline_int2_key = 2
+
+        scope.objects = {
+            IntInlineUnion.__name__: schema.ObjectType(
+                IntInlineUnion,
+                {
+                    "union": schema.PropertyType(
+                        schema.OneOfIntType(
+                            {
+                                inline_int_key: schema.RefType(
+                                    InlineInt.__name__, scope
+                                ),
+                                inline_int2_key: schema.RefType(
+                                    InlineInt2.__name__, scope
+                                ),
+                            },
+                            scope,
+                            discriminator_inlined=True,
+                            discriminator_field_name=discriminator_field_name,
+                        )
+                    )
+                },
+            ),
+            InlineInt.__name__: schema.ObjectType(
+                InlineInt,
+                {
+                    "msg": schema.PropertyType(schema.StringType()),
+                    "code": schema.PropertyType(schema.IntType()),
+                    discriminator_field_name: schema.PropertyType(
+                        schema.IntType()
+                    ),
+                },
+            ),
+            InlineInt2.__name__: schema.ObjectType(
+                InlineInt2,
+                {
+                    "msg2": schema.PropertyType(schema.StringType()),
+                    discriminator_field_name: schema.PropertyType(
+                        schema.IntType()
+                    ),
+                },
+            ),
+        }
+
+        defs = schema._JSONSchemaDefs()
+        json_schema = scope._to_jsonschema_fragment(scope, defs)
+        self.assertEqual(
+            {
+                "$defs": {
+                    IntInlineUnion.__name__: {
+                        "type": "object",
+                        "properties": {
+                            "union": {
+                                "oneOf": [
+                                    {
+                                        "$ref": (
+                                            f"#/$defs/{InlineInt.__name__}"
+                                            "_discriminated_int_"
+                                            f"{inline_int_key}"
+                                        )
```

Comment:
I would be inclined to code this with concatenation instead of (at least on of the) f-strings:
```python
                                        "$ref": (
                                            "#/$defs/" + InlineInt.__name__
                                            + "_discriminated_int_"
                                            + inline_int_key
                                        )
```

Ditto lines 2232, 2307, and 2314.

Reply:
won't do